mycore-iiif/pom.xml

```diff
@@ -29,5 +33,9 @@
       <groupId>org.mycore</groupId>
       <artifactId>mycore-base</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.mycore</groupId>
+      <artifactId>mycore-restapi</artifactId>
+    </dependency>
```

Comment:
Should we avoid this dependency? - We should keep the components as separate as possible.
Is it an option to move  org.mycore.restapi.MCRJerseyRestApp (with a different name/package) to mycore-base?
There must be some code dealing with Jersey-Resources.